Documentation/dev-internal/release.md

```diff
@@ -2,7 +2,7 @@
 
 The guide talks about how to release a new version of etcd.
 
-The procedure includes some manual steps for sanity checking but it can probably be further scripted. Please keep this document up-to-date if making changes to the release process. 
+The procedure includes some manual steps for sanity checking but it can probably be further scripted. Please keep this document up-to-date if making changes to the release process.
```

Comment:
`The procedure includes some manual steps for sanity checking but it can probably be further scripted.`
=>
`The procedure includes some manual steps for sanity checking, but it can probably be further scripted.`